packages/cli/src/databases/entities/WebhookEntity.ts

```diff
@@ -21,4 +21,26 @@ export class WebhookEntity {
 
 	@Column({ nullable: true })
 	pathLength?: number;
+
+	/**
+	 * Unique path section of production webhook URL, excluding instance URL and base `webhook/` segment.
+	 */
+	private get uniquePath() {
+		return this.webhookPath.includes(':') ? this.workflowId + this.webhookPath : this.webhookPath;
+	}
+
+	get cacheKey() {
+		return `cache:webhook:${this.method}-${this.uniquePath}`;
```

Comment:
hard coding the cache: here will result in n8n:cache:cache:webhook on redis. will work of course, but a bit redundant...

Reply:
Is this documented somewhere? Doing the same in the roles PR and Adi/Omar didn't comment on this.